content/concept.md

```diff
@@ -86,11 +86,11 @@ One such type is called a **Triple Pattern Fragment**
 It consists of:
 
 - **data** that corresponds to a triple pattern
-  _([example](http://data.linkeddatafragments.org/dbpedia?subject=&predicate=rdf%3Atype&object=dbpedia-owl%3ARestaurant))_.
+  _([example](https://data.linkeddatafragments.org/dbpedia?subject=&predicate=rdf%3Atype&object=dbpedia-owl%3ARestaurant))_.
```

Comment:
All of the links to TPF instances can remain linked to http. (unless you have a different opinion on this @RubenVerborgh?)